pkg/templater/templater.go

```diff
@@ -0,0 +1,76 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+// Package templater allows to override default Cobra usage template with grouped commands.
+package templater
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+type CommandGroup struct {
+	Message  string
+	Commands []*cobra.Command
+}
+
+type CommandGroups []CommandGroup
+
+// Add adds grouped commands to a specific Cobra command.
+func (g CommandGroups) Add(c *cobra.Command) {
+	for _, group := range g {
+		c.AddCommand(group.Commands...)
+	}
+}
+
+// SetUsageTemplate sets usage template with command groups for a specific Cobra command.
+func (g CommandGroups) SetUsageTemplate(c *cobra.Command) {
+	cobra.AddTemplateFunc("commandGroups", func(cmd *cobra.Command) string {
+		groups := []string{}
+		for _, cmdGroup := range g {
+			cmds := []string{cmdGroup.Message}
+			for _, cmd := range cmdGroup.Commands {
+				if cmd.IsAvailableCommand() {
+					cmds = append(cmds, "  "+rpad(cmd.Name(), cmd.NamePadding())+"   "+cmd.Short)
+				}
+			}
+			groups = append(groups, strings.Join(cmds, "\n"))
+		}
+		return strings.Join(groups, "\n\n")
+	})
+	c.SetUsageTemplate(usageTemplate)
+}
+
+// rpad adds padding to the right of a string.
+func rpad(s string, padding int) string {
```

Comment:
Can you add some basic test case for `rpad()` (or more) in order to have at least _some_ code coverage for the new package?

```console
$ make test | grep templater
?       reanahub/reana-client-go/pkg/templater  [no test files]
```